invenio_rdm_records/cli.py

```diff
@@ -39,6 +42,86 @@ def demo():
     click.secho('Created records!', fg='green')
 
 
+@rdm_records.command('demo_submission_requests')
+@click.option("-u", "--user", type=click.STRING)
+@with_appcontext
+def demo_submission_requests(user=system_identity.id):
+    """Create 100 fake submission requests for demo purposes."""
+    click.secho('Creating demo requests...', fg='green')
+
+    # Create 25 drafts in each community.
+    n_drafts = 25
+    # Create 4 communties.
+    for _ in range(4):
+        fake_community = create_fake_community()
+        fake_drafts = []
+        for _ in range(n_drafts):
+            fake_drafts.append(create_fake_draft())
+        create_demo_community_submission_request.delay(
+            user,
```

Comment:
IMO, the user should be hardcoded here, not passed as a param. As a dev/user, I have no idea what user to pass.